plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py

```diff
@@ -101,23 +109,39 @@ def model_loader(self, model_name: str, **kwargs):
         )
         AutoModelForCausalLM.from_config = _from_config  # patch
 
+        if self.use_external_lib:
+            kwargs = {
+                "low_cpu_mem_usage": low_cpu_mem_usage,
+                "use_marlin": False,  # disable, cannot be used for training (no forward+backward)
+                "disable_exllama": True,  # disable, cannot be used for training (no backward)
+                "use_tritonv2": True,
+                "trainable": True,  # only support trainable mode
+            }
+        else:
+            kwargs = {
+                "low_cpu_mem_usage": low_cpu_mem_usage, # this is only used for device map
+                "backend": Backend.TRITON,
+            }
+
+
         # this is a HF method that checks if the low_cpu_mem mode is enabled
         # via HF accelerate
         if is_fsdp_enabled():
-            # Local
-            from .autogptq_utils import (  # pylint: disable=import-outside-toplevel
-                _patch_target_module,
-                make_sure_no_tensor_in_meta_device,
-            )
-
-            # We patch `make_sure_no_tensor_in_meta_device`
-            # from autogptq to avoid errors on models without bias
-            _patch_target_module(
-                to_patch="auto_gptq.modeling._utils.make_sure_no_tensor_in_meta_device",
-                replace_with=make_sure_no_tensor_in_meta_device,
-                target_module="auto_gptq.modeling._base",
-            )
-            low_cpu_mem_usage = True
+            if self.use_external_lib:
+                # Local
+                from .autogptq_utils import (  # pylint: disable=import-outside-toplevel
+                    _patch_target_module,
+                    make_sure_no_tensor_in_meta_device,
+                )
+
+                # We patch `make_sure_no_tensor_in_meta_device`
+                # from autogptq to avoid errors on models without bias
+                _patch_target_module(
+                    to_patch="auto_gptq.modeling._utils.make_sure_no_tensor_in_meta_device",
+                    replace_with=make_sure_no_tensor_in_meta_device,
+                    target_module="auto_gptq.modeling._base",
+                )
+                kwargs["low_cpu_mem_usage"] = True
```

Comment:
it is strange that this is passed in only for the `external_lib` path. The extracted path should also support `low_cpu_mem_usage`